### ISSUE TITLE
deno run -A -r https://fresh.deno.dev does not accept single Enter on project name to pick default fresh-project

### ISSUE THREAD

**Issue by king8fisher** [denoland/deno#21886]
Windows 11
Terminal App (PowerShell)

Upon clicking Enter, project initializer seems to keep waiting for input, and if you enter the project name after a bunch of failed new lines, throws

```
error: Uncaught (in promise) Error: The filename, directory name, or volume label syntax is incorrect. (os error 123): rfresh-project'rs\[removed]\Desktop\king8fisher\bug-report-deno-fresh-fragment\
  const dir = [...Deno.readDirSync(resolvedDirectory)];
                       ^
    at Object.readDirSync (ext:deno_fs/30_fs.js:170:14)
    at https://deno.land/x/fresh@1.6.1/init.ts:76:24
```

Another funny thing is addition of `r`s in the error message as if it tries to get rid of `\r\n` and remains with `r` - this could be the source of the issue (potentially).

Suggestion: offer `deno-fresh-project` instead.

**Comment by kjloveless** [comment]
https://github.com/denoland/deno/issues/21812

related issue?

**Comment by bartlomieju** [comment]
Fixed by https://github.com/denoland/deno/pull/21893, it will be released in Deno v1.39.3 tomorrow.
